tests/test_ext_intersphinx.py

```diff
@@ -566,3 +568,79 @@ def test_intersphinx_role(app, warning):
 
     # explicit title
     assert html.format('index.html#foons') in content
+
+
+empty_inventory_v1 = b'''\
+# Sphinx inventory version 1
+# Project: foo
+# Version: 1.0
+'''
+
+
+empty_inventory_v2 = b'''\
+# Sphinx inventory version 2
+# Project: foo
+# Version: 2.0
+# The remainder of this file is compressed with zlib.
+''' + zlib.compress(b'')
```

Comment:
Hardcode the value of zlib.compress(..) and put a comment.

Other suggestion: make these constants private or create a class testing `find_replacements` so that the logic is "contained" (with a class-based approach you could put the setup of your test in a method).